build.gradle

```diff
@@ -97,7 +97,7 @@ dependencies {
     // kafka & zookeeper
     compile 'org.apache.kafka:kafka-clients:0.9.0.0'
     compile 'org.apache.kafka:kafka_2.11:0.9.0.0'
-    compile 'org.apache.zookeeper:zookeeper:3.4.5'
+    compile 'org.apache.curator:curator-framework:2.9.1'
```

Comment:
@CyberDem0n  BTW, will it still be possible to use pure Zookeeper without Exhibitor?